apps/fluent-tester/src/FluentTester/theme/applyBrand.ts

```diff
@@ -0,0 +1,102 @@
+import { testerTheme } from './CustomThemes';
+
+export type OfficeBrand = 'Default' | 'Office' | 'Word' | 'Excel' | 'Powerpoint' | 'Outlook';
+type BrandRampKey = 'App1' | 'App2' | 'App3' | 'App4' | 'App5' | 'App6' | 'App7' | 'App8';
+type BrandRamps = { [K in OfficeBrand]: { [J in BrandRampKey]: string } };
+
+const brandColors: BrandRamps = {
+  /**
+   * default brand just passes through what is in the theme
+   * */
+  Default: undefined,
+  /**
+   * Office brands, applied via a small App color ramp
```

Comment:
Why are we prioritizing Office brands when this has plenty of non-Office clients? Are all clients expected to add their branding constants here...?